### PR TITLE
SASS-48

### PR DESCRIPTION
**Pull request by TheDeadlyPianist**

### CODE REVIEW

File: app/common/SessionValues.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+object SessionValues {
+  val CLIENT_MTDITID = "MTDITID"
+}
```
Comment (juamalBlackman) [inline app/common/SessionValues.scala:21]: Not currently used at the moment

File: build.sbt [hunk 2]
```diff
@@ -1,10 +1,37 @@
...
+    ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
+    ScoverageKeys.coverageMinimum := 95,
+    ScoverageKeys.coverageFailOnMinimum := true,
+    ScoverageKeys.coverageHighlighting := true
+  )
+}
+
 lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .settings(
```
Comment (juamalBlackman) [inline build.sbt:37]: .settings(coverageSettings: _*) is missing and is coverage staying at 95 or going to 90

File: app/models/User.scala [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.mvc.{Request, WrappedRequest}
+
+case class User[T](mtditid: String, arn: Option[String])(implicit request: Request[T]) extends WrappedRequest[T](request) {
+  def isAgent: Boolean = arn.nonEmpty
+}
```
Comment (juamalBlackman) [inline app/models/User.scala:23]: needs to be tested 
Screenshot 2020-10-29 at 2 19 18 pm

**Review by juamalBlackman — CHANGES_REQUESTED** [review]

**Comment by juamalBlackman** [comment]
might be worth also disabling Junit plugins to prevent unstable builds 
  .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
  .disablePlugins(JUnitXmlReportPlugin)

**Review by matt712 — COMMENTED** [review]
There are some old dependencies in the AppDependencies.scala file.  https://catalogue.tax.service.gov.uk/service/income-tax-dividends

**Review by matt712 — APPROVED** [review]

**Pull request merged**